user/languages/csharp.md

```diff
@@ -121,6 +121,28 @@ script:
   - mono ./testrunner/xunit.runners.1.9.2/tools/xunit.console.exe ./MyPoject.Tests/bin/Release/MyProject.Tests.dll
 {% endhighlight %}
 
+#### Using solution-level nuget package
+Another way is to add the console testrunner of your choice as a solution-level nuget package.
+
+For many .net projects this will be the file found at:
+
+  ./.nuget/packages.config
+
+nuget restore solution-name.sln will then install that package as well.
+
+{% highlight yaml %}
+language: csharp
+solution: solution-name.sln
+install:
+  - nuget restore solution-name.sln
+script:
+  - xbuild /p:Configuration=Release solution-name.sln
+  - mono ./testrunner/xunit.runners.*/tools/xunit.console.exe ./MyPoject.Tests/bin/Release/MyProject.Tests.dll
+
+{% endhighlight %}
+
+Notice the use of filename expansion (the *) in order to avoid having to hard code the version of the test runner
```

Comment:
wrap the `*` in backticks so it is properly escaped. the sentence also misses a `.` at the end.